tests/Aspire.Hosting.Tests/Containers/ContainerResourceBuilderTests.cs

```diff
@@ -80,7 +79,7 @@ public void WithImageTagThrowsIfNoImageAnnotation()
         using var builder = TestDistributedApplicationBuilder.Create();
         var container = builder.AddResource(new TestContainerResource("testcontainer"));
 
-        var exception = Assert.Throws<InvalidOperationException>(() => container.WithImageTag(RedisContainerImageTags.Tag));
+        var exception = Assert.Throws<InvalidOperationException>(() => container.WithImageTag("7.1"));
```

Comment:
Not a fan of having the same number in several places, as that makes it hard to update. What was wrong with the previous approach where the version was in a single place?

Reply:
This test doesn't need to use redis. I can make it a variable, it doesn't really matter.